scripts/build_addon.sh

```diff
@@ -0,0 +1,64 @@
+# !/bin/bash
+
+if [ "$#" -eq 0 ]; then
+	echo "usage build_addon <product path> <release/debug - optional>"
+	exit
+fi
+
+PRODUCT_ROOT_PATH=$1
+BUILD_CONF=
+ADDON_PATH=$PRODUCT_ROOT_PATH/node_addons/FormatConverter
+FORMAT_CONVERTER_BIN=FormatConverter.so
+OS=`uname`
+echo "OS=$OS"
+
+[ "$#" -eq 2 ] && BUILD_CONF=$2 || BUILD_CONF=Release
+
+if [ "$BUILD_CONF" = "debug" ]; then
+	BUILD_CONF=Debug
+	echo "Debug mode"
+else
+	BUILD_CONF=Release
+	echo "Release mode"
+fi
+
+echo "ADDON_PATH=$ADDON_PATH"
+
+pushd $ADDON_PATH
+
+	`which node-gyp` || npm install node-gyp -g
+
+	[ -d '/usr/local/lib/node_modules/nan' ] || npm install nan -g
```

Comment:
Never presume the global NPM modules dir is /usr/local/lib/node_modules.
Use:
```
npm list --depth 1 -g nan
```
and check the RC.